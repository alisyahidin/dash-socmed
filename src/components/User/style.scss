.user:hover {
  text-decoration: none;
}