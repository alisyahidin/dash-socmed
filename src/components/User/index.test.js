import React from 'react'
import { shallow } from 'enzyme'
import User from './index'

it('render <User /> correctly', () => {
  const component = shallow(<User />)

  expect(component).toMatchSnapshot()
})