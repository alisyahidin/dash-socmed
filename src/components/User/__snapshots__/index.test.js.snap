// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <User /> correctly 1`] = `
<Link
  className="user d-flex align-items-center my-sm-2"
  replace={false}
  to="/profile"
>
  <withAvatarConfig
    className="mr-sm-2"
    round={true}
    size="40px"
  />
  <h4 />
</Link>
`;
